### PR TITLE
PSIRT/VA fix December for vpc block csi driver 4.3

### PR DESCRIPTION
**Pull request by GunaKKIBM**

### CODE REVIEW

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 1]
```diff
@@ -28,20 +28,22 @@
...
+                    "Fixes CVE-2022-42898 and PSIRT PVR0406812"
+                 ]
             },
             "previous-revision": {
-                "revision": "10",
-                "release-date": "07 November 2022",
-                "cves": "CVE-2022-37434,CVE-2020-35525,CVE-2020-35527,CVE-2022-3515,CVE-2022-2509,CVE-2022-40674",
+                "revision": "11",
+                "release-date": "17 November 2022",
+                "cves": "CVE-2022-42898",
```
Comment (ashimagarg27) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Fix "cves" key in "previous-revision":
expected value:
     "cves": "CVE-2022-1304,CVE-2016-3709"

**Review by ashimagarg27 — APPROVED** [review]

**Comment by GunaKKIBM** [comment]
Closing this PR as 4.3 vpc block csi driver template is deprecated.

**Comment by GunaKKIBM** [comment]
Re opened the PR - as 4.3 should still be supported on satellite.

**Review by prankulmahajan — APPROVED** [review]
lgtm

**Review by nkkashyap — APPROVED** [review]
lgtm

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 2]
```diff
@@ -28,22 +28,24 @@
         "4.3.0": {
             "status": "deprecated",
             "latest-revision": {
+                "revision": "12",
+                "release-date": "09 January 2023",
+                "cves": "CVE-2022-42898",
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: `"cves": "CVE-2022-42898",` ----->
`"cves": "CVE-2022-42898,CVE-2022-41717,CVE-2022-41720",`

and 

`"fixes": [
                    "Fixes CVE-2022-42898,CVE-2022-41717,CVE-2022-41720 and PSIRT PVR0406812,PVR0410879,PVR0410611"
                 ]`
-----> `"fixes": []`

**Review by mssachan — CHANGES_REQUESTED** [review]
LGTM. Just a minor change required

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**